static/app/views/alerts/rules/metric/incompatibleAlertQuery.tsx

```diff
@@ -123,42 +122,6 @@ export function IncompatibleAlertQuery(props: IncompatibleAlertQueryProps) {
   eventTypeDefault.query += ' event.type:default';
   const eventTypeErrorDefault = props.eventView.clone();
   eventTypeErrorDefault.query += ' event.type:error or event.type:default';
-  const pathname = `/organizations/${props.orgSlug}/discover/results/`;
-
-  const eventTypeLinks = {
```

Comment:
I assume the intention was to add these as options below the error message to "Use xyz event type, but there were issues with generating the URL. I think the problem was that the project param in the alerts page uses the slug, whereas `generateQueryStringObject` expects IDs. The resulting URLs had `NaN`s in them

I decided to drop these because they aren't being rendered currently anyways